python/python/tests/test_transaction.py

```diff
@@ -60,8 +60,16 @@ async def transaction_test(
     value = datetime.now(timezone.utc).strftime("%m/%d/%Y, %H:%M:%S")
     value2 = get_random_string(5)
     value3 = get_random_string(5)
+    lib_name = f"mylib{get_random_string(5)}"
+    code = f"#!lua name={lib_name}\n redis.register_function('myfun{get_random_string(5)}', function(keys, args) return args[1] end)"
     args: List[TResult] = []
 
+    if not await check_if_server_version_lt(redis_client, "7.0.0"):
+        transaction.function_load(code)
```

Comment:
Nit: add flush before and after, otherwise test will fail if you restart it without recreating the cluster
You can use custom command until you replace it with the implementation

Reply:
I use `get_random_string(5)` so it should be fine